dato.config.js

```diff
@@ -1,23 +1,31 @@
 // @flow
 
-const filterByKeys = (data, filterKeys, areIncluded) =>
+const filterByKeys = (data, filterKeys, areIncluded, isTranslatedPart) =>
```

Comment:
I'm not sure about the naming of this new parameter. What about `excludeEmpty` or something similar?